Assignment-3/app/src/components/SuccessLogin.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const SuccessLogin = (props) => {
+  return (
+    <div>
+      <p>Welcome! {props.username}</p>
+    </div>
+  )
+}
```

Comment:
What is the use of this component and where we are using it

Reply:
i can remove unwanted componnet

Reply:
Yes Please remove the unused files and component as it makes difficult to review